### DISCUSSION TITLE
data/2023/05/27/pandas-ai/

### DISCUSSION THREAD

**Discussion by giscus** [zzsza/blog-comments#142] [Comments]
# data/2023/05/27/pandas-ai/

Python으로 데이터 분석을 할 때 빼먹을 수 없는 라이브러리인 Pandas. 아마 많은 분들이 사용하시고, 처음에 많이 학습하고 있을거에요 최근 Pandas-AI라는 라이브러리가 공개되었는데 대화형으로 Pandas를 사용할 수 있는 시대가 되었습니다. 아직 버전은 0.2지만, 추후에 더 많이 발전할 것 같네요 참고 : 이 글은 0.2.2 버전을 기준으로 작성했고, 글을 보시는 시기에 따라...

https://zzsza.github.io/data/2023/05/27/pandas-ai/

**Comment by horu2day** [comment]
혹시 표에 글씨가 한글로 나올수 있게 하는 방법 아시나요? Python 에러에서 한글폰트를 깔아주라고 하는데 이미깔린 폰트이고, 뭘 어케해야 되는지 좀 알려주세용???
